packages/atlas/src/components/_inputs/TokenInput/TokenInput.tsx

```diff
@@ -37,7 +37,7 @@ const _TokenInput: ForwardRefRenderFunction<HTMLInputElement, TokenInputProps> =
       {...rest}
       ref={ref}
       type="number"
-      nodeStart={<JoyTokenIcon variant="gray" size={24} />}
+      nodeStart={<JoyTokenIcon variant="gray" size={16} />}
```

Comment:
This isn't desired in every place that uses `TokenInput`. For example, in NFT purchase view, the icon has 24 px

Reply:
It seems that it isn't correct anymore

`(Breaking change!) Changed the type of icons used in the component from Controls (24px) to Action (16px),`
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=7058%3A174881

Reply:
You are right, thanks for pointing out